base/src/com/google/idea/blaze/base/projectview/section/sections/ImportSection.java

```diff
@@ -50,7 +53,14 @@ protected WorkspacePath parseItem(
       if (parser.isRecursive()) {
         File projectViewFile = parseContext.getWorkspacePathResolver().resolveToFile(workspacePath);
         if (projectViewFile != null) {
-          parser.parseProjectView(projectViewFile);
+          boolean projectViewImportsMandatory = !Registry.is("bazel.projectview.optional.imports");
+          if (projectViewFile.exists() || projectViewImportsMandatory) {
```

Comment:
I'd probably flip the argument for (a) readability though this is imo (b) less file operations as the value will be `true` by default.